canonical-web-and-design/ubuntu.com
# Remove deprecated components

## Done

- removed `_pattern_inline-images.scss`, as Vanilla now handles everything those styles were doing.
- updated renamed `u-hide` classes.
- removed `u-float` class, as it's now part of Vanilla.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Visit the following pages and ensure that the header image is hidden on mobile:
  - [x] /blog/topics/design
  - [x] /blog/topics/juju
  - [x] /blog/topics/maas
  - [x] /blog/topics/robotics
  - [x] /blog/topics/snappy
  - [x] /desktop/developers
  - [x] /desktop/features

- Visit the following pages and ensure that the header images are floated to the right:
  - [x] /containers

- Visit the following page, click "I am signing as an individual contributor.", and see that a previously hidden license agreement appears:
  - [x] /legal/contributors/agreement

## Issue / Card

Fixes #5648
